Code/GraphMol/RGroupDecomposition/RGroupDecomp.cpp

```diff
@@ -827,27 +879,31 @@ struct RGroupDecompData {
     for (auto &it : best) {
       for (auto rit = it.rgroups.begin(); rit != it.rgroups.end(); ++rit) {
         if (rit->first >= 0) userLabels.insert(rit->first);
```

Comment:
turns out I don't directly use userLabels.  I'm finding this bit of code could be cleaned up a bit.  However, we do need to keep track of so called indexLabels which probably should be renamed UnassignedLabels which might make more sense.